drivers/Watchdog.h

```diff
@@ -46,21 +47,24 @@ namespace mbed {
  */
 class Watchdog {
 public:
-    Watchdog() {}
-
+    Watchdog(uint32_t timeout = 1, const char *const str = NULL);
+    ~Watchdog();
 public:
+
     /** Start an independent watchdog timer with specified parameters
      *
-     *  @param timeout      Timeout of the watchdog in milliseconds
-     *
-     *  @return status WATCHDOG_STATUS_OK if the watchdog timer was started
-     *                 successfully. WATCHDOG_INVALID_ARGUMENT if one of the input
-     *                 parameters is out of range for the current platform.
-     *                 WATCHDOG_NOT_SUPPORTED if one of the enabled input
-     *                 parameters is not supported by the current platform.
+     * Assert for multiple calls of start
      */
-    watchdog_status_t start(const uint32_t timeout);
+    void start();
```

Comment:
why these methods were changed to "void method(...)" ?

Reply:
Please refer PR#9243 review comments and new design changed this to void